### PR TITLE
Feat: Login auth and Context for liked video

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Feat: Login auth](https://github.com/pritam-kr/playme/commit/7b2c11e182e441b54ce3eb6d25d1d4404d79475e)
- [Feat: context for liked video](https://github.com/pritam-kr/playme/commit/2ccea3561001fa962f6b20727ddeefc05c681deb)

- Created more pages 
  - History
  - Watch Later
  - Page for liked video
  
- Created Helper function logged message
- Created context and reducer for liked video
- did initial fetching for liked video with help of token
- provided private route

### CODE REVIEW

File: src/Component/Topbar/Topbar.jsx [hunk 1]
```diff
@@ -3,9 +3,14 @@ import * as FaIcons from "react-icons/fa";
 import * as BiIcons from "react-icons/bi";
 import "./Topbar.css";
 import { useNavigate } from "react-router-dom"
+import {useAuthContext} from "../../Context/Index"
 
 const Topbar = () => {
   const navigate = useNavigate()
+  const {isAuth} = useAuthContext()
+  
+  // console.log(isAuth)
```
Comment (horsemaker) [inline src/Component/Topbar/Topbar.jsx:12]: You could remove this comment from the code.

**Review by horsemaker — APPROVED** [review]
Everything looks fine Pritam. 👌
Nicely done! 👍

File: src/App.js [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
-        <Route path="/playlist" element={<Playlist />} > </Route> 
+        {!isAuth && <Route path="/login" element={<Login />} > </Route>}
+        <Route path="/playlist" element={<PrivateRoute><Playlist /></PrivateRoute>} > </Route> 
         <Route path="/video/:videoId" element={<SingleVideo />} > </Route>
         <Route path="/mockman" element={<MockAPI />} > </Route>
+        <Route path="/history" element={<PrivateRoute><History /></PrivateRoute>} > </Route>
+        <Route path="/likes" element={<PrivateRoute><Likes /></PrivateRoute>} > </Route>
+        <Route path="/watchlater" element={<PrivateRoute><WatchLater /></PrivateRoute>} > </Route>
+        <Route path="/*" element={<Navigate  to="/explore" />}></Route>  
       </Routes>
```
Comment (srishti-maurya) [inline src/App.js:40]: Suggestion: You may create a Routes Component here to make App.js more cleaner

**Review by srishti-maurya — APPROVED** [review]
Liked the use of comments and barrel exports and the toast library used. Great work

**Pull request merged**